pkg/cli/haproxy.go

```diff
@@ -107,6 +145,7 @@ listen psql
     bind :26257
     mode tcp
     balance roundrobin
-{{range .}}    server cockroach{{.Desc.NodeID}} {{.Desc.Address.AddressField}} check
+    option httpchk GET /health?ready=1
```

Comment:
Does `httpchck` work for HTTPS endpoints automatically or do you need to tweak this for secure clusters?

Reply:
Tested it with a secure cluster and it works fine.